### ISSUE TITLE
"save as" for png format does not work as it should

### ISSUE THREAD

**Issue by EJWagenmakers** [jasp-stats/jasp-desktop#1837]
When I copy a figure, it is in standard png format and it works fine. 
When I select "save as" and then indicate the png format, the plot does not have the right background. I think this occurs because with "save as" the plot is highlighted, but I'm not sure. Attached are two figures: the good one obtained using the old method (simply "copy") and the poor one obtained using "save as".
![correlationmatrix](https://user-images.githubusercontent.com/8603015/27625322-893a278c-5be4-11e7-9415-75572aa6ffaa.png)
<img width="750" alt="correlationmatrix2" src="https://user-images.githubusercontent.com/8603015/27625324-8b9db26e-5be4-11e7-95c2-730d16abe85e.png">

**Comment by EJWagenmakers** [comment]
Darn, they both look good on GitHub! Not on my computer. Drop by if you want to see for yourself.
E.J.

**Comment by TimKDJ** [comment]
The first figure has a white background, while the second is transparant.
Because Github is white they both look okay.

You can see the difference when you open them in separate tabs:
[one](https://user-images.githubusercontent.com/8603015/27625322-893a278c-5be4-11e7-9415-75572aa6ffaa.png)
[two](https://user-images.githubusercontent.com/8603015/27625324-8b9db26e-5be4-11e7-95c2-730d16abe85e.png)

I agree that we should make this uniform. Not sure what's more practical to a user, white or transparant...

**Comment by EJWagenmakers** [comment]
What does the eps do? Transparent is not what I would expect.

**Comment by TimKDJ** [comment]
No idea, do you know @vankesteren?

**Comment by AlexanderLyNL** [comment]
Transparant 

<img width="453" alt="screen shot 2017-06-28 at 10 09 52" src="https://user-images.githubusercontent.com/10271675/27626873-fee3bcfa-5be9-11e7-941b-0b61d0332212.png">

**Comment by EJWagenmakers** [comment]
Can we google this? Or else a twitter poll :-)

********************************************
Eric-Jan Wagenmakers
Department of Psychological Methods, room G 0.29
University of Amsterdam, Nieuwe Achtergracht 129B
Letter: PO Box 15906, 1001 NK Amsterdam
Parcel: Valckenierstraat 59,  1018 XE Amsterdam

Web: ejwagenmakers.com
Book: bayesmodels.com
Stats: jasp-stats.org
Email: EJ.Wagenmakers@gmail.com
Cell: (+31) 6 45626624

“Man follows only phantoms.”
Pierre-Simon Laplace, last words
********************************************

**Comment by AlexanderLyNL** [comment]
Twitter poll

**Comment by vankesteren** [comment]
Yes transparent is what it does. Would be a relatively simple fix to make it have a white background! It's a design decision.

**Comment by EJWagenmakers** [comment]
OK, but what does the eps/tiff do? Would be weird to have a png that's
transparent and an eps/tiff that is not. I personally dislike transparent.
E.J.

********************************************
Eric-Jan Wagenmakers
Department of Psychological Methods, room G 0.29
University of Amsterdam, Nieuwe Achtergracht 129B
Letter: PO Box 15906, 1001 NK Amsterdam
Parcel: Valckenierstraat 59,  1018 XE Amsterdam

Web: ejwagenmakers.com
Book: bayesmodels.com
Stats: jasp-stats.org
Email: EJ.Wagenmakers@gmail.com
Cell: (+31) 6 45626624

“Man follows only phantoms.”
Pierre-Simon Laplace, last words
********************************************

**Comment by EJWagenmakers** [comment]
People *expect* the plots to be non-transparent. This is also pleasant if you want to copy-paste them in PowerPoint. So let's set them to non-transparent by default, but allow them to change this setting through an option (to be implemented later) in the general preferences.

Cheers,
E.J.

**Comment by AlexanderLyNL** [comment]
I'm only bumping this to high priority so we will make a decision about this.
